mj_envs/envs/env_base.py

```diff
@@ -13,7 +13,7 @@
 from torchvision.models import resnet50, ResNet50_Weights, resnet34, ResNet34_Weights, resnet18, ResNet18_Weights
 import torchvision.transforms as T
 
-from envs.obj_vec_dict import ObsVecDict
+from .obj_vec_dict import ObsVecDict
```

Comment:
Aha, thanks for catching this. I moved this a few days ago.
Are you okay with it being `mj_envs.envs.obj_vec_dict`? 
I know, this makes it independent of the working directory user is on. Thoughts?